sdk/src/types/block/output/nft.rs

```diff
@@ -487,6 +487,14 @@ impl StateTransitionVerifier for NftOutput {
     }
 }
 
+impl WorkScore for NftOutput {
```

Comment:
I'm a bit annoying but when there is a StorageScore impl, can you put it next to it please?

Reply:
Done